src/app/pages/apps/components/chart-wizard/chart-wizard.component.ts

```diff
@@ -116,6 +118,11 @@ export class ChartWizardComponent implements OnInit, AfterViewInit, OnDestroy {
           nextElement = document.getElementById(id);
           if (idx === path.length - 1) {
             nextElement?.scrollIntoView({ block: 'center' });
+            nextElement.classList.add('highlighted');
+
+            of(([])).pipe(delay(999))
```

Comment:
We can replace `of(([])).pipe(delay(999))` with `timer(999)`. `timer` will wait for 999 milliseconds before emitting 0.
https://rxjs.dev/api/index/function/timer

Reply:
ofc